package.json

```diff
@@ -10,7 +10,7 @@
     "minVersion": "57.0",
     "maxVersion": "*",
     "multiprocessCompatible": true,
-    "hasEmbeddedWebExtension": false,
+    "hasEmbeddedWebExtension": true,
```

Comment:
Hmmm... I should probably just revert this later on. It was part of an experiment I did earlier.